source/server/admin/stats_request.cc

```diff
@@ -77,34 +51,30 @@ bool StatsRequest::nextChunk(Buffer::Instance& response) {
   while (response.length() - starting_response_length < chunk_size_) {
     while (stat_map_.empty()) {
       if (phase_stat_count_ == 0) {
-        render_->noStats(response, phase_string_);
+        getRender().noStats(response, phase_labels_[phases_.at(phase_index_)]);
```

Comment:
perf nit: hoist the render object into a temp outside the outer `while` avoid calling the same virtual call 3x per iteration.